vhdbuilder/packer/install-dependencies.sh

```diff
@@ -347,7 +347,7 @@ done
 CPA_IMAGES="
 1.3.0
 1.3.0_v0.0.5
-1.7.1
```

Comment:
let's keep 1.7.1 during the transition period. (vmas scale up will use new VHd but still requires 1.7.1 before chart upgrade)

Reply:
yea, that makes sense